src/IQFeed.CSharpApiClient.Tests/Lookup/LookupRateLimiterTests.cs

```diff
@@ -113,7 +113,7 @@ public void Should_Validate_MaxCount()
             var lookupRateLimiter = new LookupRateLimiter(requestsPerSecond);
 
             // Assert
-            var expectedMaxCount = 25;
+            var expectedMaxCount = 50;
```

Comment:
could you please refresh  my mind. whats the real limit ?

50 messages per second ?

Reply:
yes, 50

Reply:
This is for just one client, right? If we simultaneously use multiple clients will we quickly go over the limit?